### DISCUSSION TITLE
Extra Space from QPage

### DISCUSSION THREAD

**Discussion by muoname** [quasarframework/quasar#14834] [General - Components / Directives / etc]
Hello is there a way to get rid of the extra space for the qpage?
![image](https://user-images.githubusercontent.com/89370995/200207081-4af63b8a-1940-419e-859f-3025da782bd7.png)

I'm currently building an app on ruby on rails using the quasar framework and I've put the mounted App.vue (which contains the drawer and the header) into the application.html.erb so that it will be visible in all other views, problem is that it renders the q-page on the App.vue which results in an extra white space. Any idea on how I can fix this?

**Comment by MilosPaunovic** [comment]
You probaly have some margin or padding classes attached to one of the wrapper elements, so take a look at that first.
